app/services/container_dashboard_service.rb

```diff
@@ -160,4 +161,44 @@ def heatmaps
       }
     }
   end
+
+  def node_utilization
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    metrics = VimPerformanceDaily.find_entries({:tz => @controller.current_user.get_timezone})
+    metrics = metrics.where(:resource_type => 'ExtManagementSystem', :resource_id => resource_ids)
+    metrics = metrics.where("timestamp > ?", 30.days.ago)
```

Comment:
@simon3z In general rails doesn't do the query and get the object until it needs to, in that case splitting the 
"metrics.where(:resource_type => 'ExtManagementSystem', :resource_id => resource_ids)" and "metrics = metrics.where("timestamp > ?", 30.days.ago)" wont make any difference because rails doesn't preform the query between the two calls (though i don't understand what the reason for splitting it).